main.tf

```diff
@@ -58,6 +58,7 @@ resource "google_compute_address" "ip" {
   purpose      = var.address_type == "INTERNAL" ? var.purpose : null
   network_tier = var.address_type == "INTERNAL" ? null : var.network_tier
   labels       = var.labels
+  description  = element(var.descriptions, count.index)
```

Comment:
If using `element()`, might need to wrap in `try(element(var.descriptions, count.index), null)`

```
        	Error:      	Received unexpected error:
        	            	FatalError{Underlying: error while running command: exit status 1; 
        	            	Error: Error in function call
        	            	
        	            	  on ../../../main.tf line 61, in resource "google_compute_address" "ip":
        	            	  61:   description  = element(var.descriptions, count.index)
        	            	    ├────────────────
        	            	    │ while calling element(list, index)
        	            	    │ count.index is 0
        	            	    │ var.descriptions is empty list of string
        	            	
        	            	Call to function "element" failed: cannot use element function with an empty
        	            	list.}
        	Test:       	TestIpAddressOnly
```

Reply:
Oh my bad, yes, agree. I've updated the branch. Please run the lint check again.